### DISCUSSION TITLE
2023-04-12 Weekly Hosting Meeting

### DISCUSSION THREAD

**Discussion by javiercasares** [WordPress/hosting-handbook#138] [Weekly Meetings]
# Meeting

/here <meeting type="hosting"> Who’s here to chat about hosting?

👋 Hello! This is the weekly meeting for the WordPress.org Hosting Team, who works with hosts and developers to help improve the WordPress user experience. This is a great time to ask any questions you may have, or visit Make/Hosting for more info. 

🥇 If you participate in the Five for the Future program, drop your WordPress.org username in a thread here 👉

🤗 If you're new here, welcome! You are welcome to introduce yourself if you like in a thread here 👉

-----

## Agenda

📓 ··· Here is the agenda for this meeting ···

**WordPress Community**
- Preparing for the Next Women & Nonbinary Release Squad

**Hosting Team Time**
- WebP Uploads
- WP-CLI 2.8.0
- Next events / Contributor Day

**Hosting Team Projects**
- Advanced Administration Handbook
- Host Test Runners

**Open Floor**
- Anything not in the agenda

Would anyone like to add to or change any items on the agenda?

OK, let’s get started with the WordPress Community news.

-----

### :wordpress: ··· Preparing for the Next Women & Nonbinary Release Squad ···

[Preparing for the Next Women & Nonbinary Release Squad](https://make.wordpress.org/core/2023/04/07/preparing-for-the-next-women-nonbinary-release-squad/)

It's time for a little planning for the other major releases this year. Volunteers are needed for the WordPress 6.3 squad (release target mid-July), especially those willing to dedicate time toward mentoring the next round of squad members. WordPress 6.4 will be run entirely by women and nonbinary squad, a reprise of the [historic WordPress 5.6 release, "Simone"](https://wordpress.org/news/2020/12/simone/), and training cohorts offered during 6.3 are an excellent opportunity for new contributors to prepare for the 6.4 release.

-----

Ok, let’s move on to Hosting Team time.

### :plugins: ··· WebP Uploads ···

[WebP Uploads](https://wordpress.org/plugins/webp-uploads/)

With the goal of having canonical plugins, the Performance Lab has a spin-off for the WebP Uploads. Please take a look at it.

This plugin adds WebP support for media uploads within the WordPress application. WebP images will be generated only for new uploads, pre-existing imagery will not be converted to WebP format. By default, WebP images will only be generated for JPEG uploads, only the original uploaded file will still exist as a JPEG image. All generated image sizes will exist as WebP only.

### 🤖 ··· WP-CLI 2.8.0 ···

[WP-CLI v2.8.0 release date](https://make.wordpress.org/cli/2023/04/05/wp-cli-v2-8-0-release-date/)

The v2.8.0 release of WP-CLI is scheduled to be published on 2023-05-03.

Among others a multitude of bug fixes and some new features, we’re also expecting this release to fix all known PHP 8.2 issues within WP-CLI itself.

The release candidate for this release will already be available sometime during the week prior, to allow for early testing. We will announce the availability of that release candidate in the #hosting-community channel to allow hosters to do early smoke testing.

### 🗓️ ···  Next events / Contributor Day ···

Heads up, some events are coming down the pipeline along with some Contributor Days to participate in:

- 2023-05-05, [WordCamp Barcelona](https://barcelona.wordcamp.org/2023/), Barcelona (Spain).
- 2023-06-08, [WordCamp Europe](https://europe.wordcamp.org/2023/), Athens (Greece).
- 2023-08-24, [WordCamp US](https://us.wordcamp.org/2023/), National Harbor, Maryland (USA).

Is anyone [going to a WordCamp](https://central.wordcamp.org/schedule/) and wants to lead a Hosting Table?

-----

Great, so just a look at our projects.

### 📕 ··· Advanced Administration Handbook ···

[Introducing the Advanced Administration handbook](https://make.wordpress.org/docs/2023/04/10/introducing-the-advanced-administration-handbook/)

The Advanced Administration handbook is now public and online.

- [Public Handbook](https://developer.wordpress.org/advanced-administration/)
- [GitHub Handbook](https://github.com/WordPress/Advanced-administration-handbook)

Currently, [everything pending to do is in the Issues part](https://github.com/WordPress/Advanced-administration-handbook/issues). A lot of "documentation + enhancement" tickets, and also some "documentation + good first issue" for Contributor Day.

If anybody needs help or mentoring for this project, @javiercasares is always available.

### :cloud: ··· Host Test Runners ···

You may have heard of [Host Test Results](https://make.wordpress.org/hosting/test-results/), a community-led project that helps WordPress users find the best hosts for their sites. Host Test Results provides a comprehensive list of hosting providers and detailed information about each host's performance and reliability.

We want to make sure that as many hosts as possible are included in the Host Test Results database, so that WordPress users and the #core team can benefit from this valuable resource. That's why we're asking you to check if your host is already supporting Host Test Results, and to add your host if it still needs to be included.

To check if your hoster is already supporting Host Test Results, visit the website (https://make.wordpress.org/hosting/test-results/) and search for your host's name. If your host is already included, great! You can use the information provided to make sure that you're getting the best possible performance and reliability from your site. 

If your host isn't included, don't worry, you can still help! You can just follow the instructions on the website to add your host to the database. It's quick and easy, and you'll be helping other WordPress users find the best hosts for their sites. However, if your host has not received reports in the last 25 revisions, we encourage you to check back with them.

- [Try the PHPUnit Test Runner](https://make.wordpress.org/hosting/handbook/tests/#try-the-phpunit-test-runner)
- [How to report: Creating your bot for WordPress.org](https://make.wordpress.org/hosting/handbook/tests/#how-to-report-creating-your-bot-for-wordpress-org)

-----

## Open Floor

Ok, if nothing else, let’s move on to 🤖 Open Floor. Folks are welcome to bring any topic up for discussion at this time.

-----

## Next meeting

⏰ Here are the [team's upcoming meetings](https://make.wordpress.org/meetings/#hosting):

- 2023-04-12 at 18:00 UTC (in a few hours)
- 2023-04-19 at 09:00 UTC (next week)
- 2023-04-19 at 18:00 UTC

📌 Okay, we're out of time for today, let's close out this meeting.

You are welcome to continue the conversation here in the #hosting-community channel or on Make/Hosting, at any time.

Thank you all for joining and for your contributions to WordPress!
See you at the next Hosting Team weekly meeting!</meeting>

**Comment by javiercasares** [comment]
cc/ @kittenkamala @Crixu @javiercasares @jessfrick

**Comment by jessfrick** [comment]
Five for the Future participants: 
- brettface
- chaion07
- carl-alberto
- jessibelle
